sass/blocks/wrapper/_wrapper.scss

```diff
@@ -0,0 +1,12 @@
+//Desktop screen
+.wrapper {
+  @media (min-width: 1200px) {
+    width: 100%;
+    max-width: 120rem;
+    margin: 0 auto;
+    position: relative;
+    -webkit-box-shadow: 0 1px 80px rgba(0, 0, 0, 0.3), 0 0 40px rgba(0, 0, 0, 0.1) inset;
+    -moz-box-shadow: 0 1px 80px rgba(0, 0, 0, 0.3), 0 0 40px rgba(0, 0, 0, 0.1) inset;
+    box-shadow: 0 1px 80px rgba(0, 0, 0, 0.3), 0 0 40px rgba(0, 0, 0, 0.1) inset;
+  }
```

Comment:
1) I don't think in designs we have any inner/inset shadow. 
2) we don't need vendor prefixes, autoprefixer does that for us
3) Your shadow is much stronger than in designs.
4) You're missing the background color on the body.
![image](https://user-images.githubusercontent.com/1570578/36917014-823224fe-1e4d-11e8-89f9-7e0211d3c45c.png)
![image](https://user-images.githubusercontent.com/1570578/36917038-95c6a1f2-1e4d-11e8-9ea2-6595b49c298f.png)